solutions/bio1/ParsingLists/index.tsx

```diff
@@ -1,7 +1,84 @@
-import { STitle } from "components";
+import { Link, SCodeBlock, SList, SText, STitle } from "components";
+import { brace, floor } from "solutions/util";
 
 export const ParsingLists = () => {
-  return <>
-    <STitle>Coming Soon!</STitle>
-  </>
+  return (
+    <>
+      <STitle>Observation</STitle>
+      <SText>
+        At no point does any value in the list affect any other value in the list. This means that at each point, instead of building the entire list (which would be far too slow), we can simply calculate the value at the position required.
+      </SText>
+
+      <STitle>Sample Explanation</STitle>
+      <SText>
+        The $i$th element with the description E(OE) is equal to: E[O[E[E[E[O[E[i]]]]]]].
```

Comment:
please use `$i$-th` instead of `$i$th`

Reply:
please can you surround stuff like E(OE) and E[i] with dollar signs so that it renders in LaTeX